src/Mod/PartDesign/App/FeatureHelix.cpp

```diff
@@ -77,21 +77,22 @@ Helix::Helix()
 {
     addSubType = FeatureAddSub::Additive;
 
-    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), "Helix", App::Prop_ReadOnly, "Base");
-    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), "Helix", App::Prop_ReadOnly, "Axis");
-    ADD_PROPERTY_TYPE(Pitch, (10.), "Helix", App::Prop_None, "Pitch");
-    ADD_PROPERTY_TYPE(Height, (30.0), "Helix", App::Prop_None, "Height");
-    ADD_PROPERTY_TYPE(Turns, (3.0), "Helix", App::Prop_None, "Turns");
+    const char* group = "Helix";
+    ADD_PROPERTY_TYPE(Base, (Base::Vector3d(0.0, 0.0, 0.0)), group, App::Prop_ReadOnly, "The center point of the helix' start.");
+    ADD_PROPERTY_TYPE(Axis, (Base::Vector3d(0.0, 1.0, 0.0)), group, App::Prop_ReadOnly, "The helix' direction.");
+    ADD_PROPERTY_TYPE(Pitch, (10.), group, App::Prop_None, "The axial distance between two turns.");
+    ADD_PROPERTY_TYPE(Height, (30.0), group, App::Prop_None, "The total length of the helix (not accounting for the extent of the profile)");
+    ADD_PROPERTY_TYPE(Turns, (3.0), group, App::Prop_None, "The number of turns in the helix (does not need to be a whole number).");
     Turns.setConstraints(&floatTurns);
-    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), "Helix", App::Prop_None, "LeftHanded");
-    ADD_PROPERTY_TYPE(Reversed, (long(0)), "Helix", App::Prop_None, "Reversed");
-    ADD_PROPERTY_TYPE(Angle, (0.0), "Helix", App::Prop_None, "Angle");
-    ADD_PROPERTY_TYPE(Growth, (0.0), "Helix", App::Prop_None, "Growth");
+    ADD_PROPERTY_TYPE(LeftHanded, (long(0)), group, App::Prop_None, "Determins whether the turning direction of helix follows the \"left-hand rule\", i.e. counter-clockwise when moving along its axis.");
+    ADD_PROPERTY_TYPE(Reversed, (long(0)), group, App::Prop_None, "Determins whether the helix points in the opposite direction of the reference axis.");
+    ADD_PROPERTY_TYPE(Angle, (0.0), group, App::Prop_None, "If non-zero, the profile's path wraps around a cone with this angle, instead of a cylinder. Positive values make the radius grow, nevatige shrink.");
```

Comment:
"If non-zero, " can be omitted. It is clear that it is the angle, and if zero, there is no angle at all.

Reply:
I don't think that "If non-zero" should be dropped because otherwise the rest "instead of a cylinder" doesn't make things clear.

> and if zero, there is no angle at all.

Why is an angle of 0 degree not an angle?

Reply:
> Why is an angle of 0 degree not an angle?

Like a distance of 0 is actually no distance ;-)

However, I think we found already a suitable wording. The one I use in the dialog:
"Angle of the cone that forms a hull around the helix."

Then maybe a second sentence that a positive angle make the cone wider along the helix axis. So maybe:

"Angle of the cone that forms a hull around the helix. A positive angle widens the cone along the axis."